Mathlib/RingTheory/Derivation/Basic.lean

```diff
@@ -358,6 +358,9 @@ protected def restrictScalars (d : Derivation S A M) : Derivation R A M where
   toLinearMap := d.toLinearMap.restrictScalars R
 #align derivation.restrict_scalars Derivation.restrictScalars
 
+@[simp]
+lemma coe_restrictScalars (d : Derivation S A M) : ⇑(d.restrictScalars R) = ⇑d := rfl
```

Comment:
Often we don't make these lemmas `simp`, since that means they can end up applying before the `Derivation`-specific lemmas have a chance.

Can you add `restrictScalars_apply` too, and make that `simp` instead?

Reply:
Done.